maplibre-demo/Cargo.toml

```diff
@@ -15,10 +15,10 @@ web-webgl = ["maplibre/web-webgl"]
 trace = ["maplibre/trace"]
 
 [dependencies]
-env_logger = "0.9.0"
+env_logger = "0.10.0"
 maplibre = { path = "../maplibre", version = "0.1.0", features = ["headless", "thread-safe-futures"]  }
 maplibre-winit = { path = "../maplibre-winit", version = "0.1.0"  }
 
 tile-grid = "0.3"
 
-clap = { version = "3.2.12", features = ["derive"] }
+clap = { version = "4.1.1", features = ["derive"] }
```

Comment:
I'm very much in favor of dropping clap due to frequent major version upgrades and too many features. Some simpler alternative would probably be totally enough for our demo.

Though that is probably not part of this task.

Reply:
structopt used to be my goto library, but it was based on clap, and got finally completely merged into clap.

It’s indeed a very big library, but when using simpler ones, there are rapidly small annoyances that aren’t managed.

There are options listed here https://github.com/rosetta-rs/argparse-rosetta-rs

Reply:
I'll maybe test out argh one day.

clap is just so big for simple tasks :)